Beams/MRML/vtkMRMLRTPlanNode.cxx

```diff
@@ -107,35 +104,15 @@ void vtkMRMLRTPlanNode::ReadXMLAttributes(const char** atts)
   vtkMRMLNode::ReadXMLAttributes(atts);
 
   // Read all MRML node attributes from two arrays of names and values
-  const char* attName = nullptr;
-  const char* attValue = nullptr;
-
-  while (*atts != nullptr)
-  {
-    attName = *(atts++);
-    attValue = *(atts++);
-
-    if (!strcmp(attName, "NextBeamNumber"))
-    {
-      this->NextBeamNumber = vtkVariant(attValue).ToDouble();
-    }
-    else if (!strcmp(attName, "TargetSegmentID"))
-    {
-      this->SetTargetSegmentID(attValue);
-    }
-    else if (!strcmp(attName, "DoseEngineName"))
-    {
-      this->SetDoseEngineName(attValue);
-    }
-    else if (!strcmp(attName, "RxDose"))
-    {
-      this->RxDose = vtkVariant(attValue).ToDouble();
-    }
-    else if (!strcmp(attName, "IsocenterSpecification"))
-    {
-      this->IsocenterSpecification = (IsocenterSpecificationType)(vtkVariant(attValue).ToInt());
-    }
-  }
+  vtkMRMLReadXMLBeginMacro(atts);
+  vtkMRMLReadXMLIntMacro( NextBeamNumber, NextBeamNumber);

```

Comment:
Ctrl+F still shows 47 different places with spaces following parenthesis.
Could you run a  find+replace to fix them?

Reply:
I've corrected a format.